pull-request.sh

```diff
@@ -49,6 +50,7 @@ create_pull_request() {
     TARGET="$(echo -n "${2}" | jq --raw-input --slurp ".")"  # pull request TO this target
     BODY="$(echo -n "${3}" | jq --raw-input --slurp ".")"    # this is the content of the message
     TITLE="$(echo -n "${4}" | jq --raw-input --slurp ".")"   # pull request title
+    MAINTAINER_CAN_MODIFY="$(echo -n "${5}" | jq --raw-input --slurp ".")"   # pull request title
 
     # JSON boolean
     if [[ "${5}" ==  "true" ]]; then                         # if PRs are draft
```

Comment:
The 5th argument to the function was previously for a draft, or this is a bug. Either way, this needs to be fixed.

Reply:
The 5th argument seems to still be draft, the 6th is for maintainer, and the 6th is for assignee.

Reply:
yes, right ill fix the order :) I would also easier if rewrote this in js 😆i love bash :))